src/pt/readmangas/src/eu/kanade/tachiyomi/extension/pt/readmangas/ReadMangas.kt

```diff
@@ -279,33 +262,68 @@ class ReadMangas() : HttpSource() {
                 url = "/title/${it.slug}#${it.id}"
             }
         }
-        return MangasPage(mangas, data.nextCursor != null) to (data.nextCursor ?: "")
+        return MangasPage(mangas, dto.hasNextPage) to dto.nextCursor
     }
 
-    private inline fun <reified T> Response.parseAs(): T {
-        return json.decodeFromString(body.string())
-    }
-
-    private fun String.toDate() =
-        try { dateFormat.parse(this)!!.time } catch (_: Exception) { 0L }
-
     private fun String.toStatus() = when (lowercase()) {
         "ongoing" -> SManga.ONGOING
         "hiatus" -> SManga.ON_HIATUS
         else -> SManga.UNKNOWN
     }
 
+    private fun Response.parseScriptToJson(): String? {
+        val document = asJsoup()
+        val script = document.select("script")
+            .map(Element::data)
+            .filter {
+                it.contains("self.__next_f", ignoreCase = true)
```

Comment:
I don't think it makes sense to use `ignoreCase = true` here, especially since further down you assume the variable `self.__next_f` exists.